platforms/core-configuration/file-collections/src/main/java/org/gradle/api/internal/file/collections/DefaultConfigurableFileCollection.java

```diff
@@ -824,4 +833,15 @@ public Configurer from(Object... paths) {
             return this;
         }
     }
+
+    @Override
+    public CompoundAssignmentStandIn toCompoundOperand() {
+        return new CompoundAssignmentStandIn();
+    }
+
+    public class CompoundAssignmentStandIn {
+        public Object plus(FileCollectionInternal rhs) {
```

Comment:
❔ Is this method called by Groovy under the hood? Should we add a comment about this or a reference to a place where this is explained?

Reply:
> Is this method called by Groovy under the hood?

It is, I've added some breadcrumbs for readers not well-versed in Groovy.